sdks/go/prepare_go_version.sh

```diff
@@ -59,8 +75,7 @@ GOBIN=$GOPATH/bin
 GOHOSTOS=`go env GOHOSTOS`
 GOHOSTARCH=`go env GOHOSTARCH`
 
-# Outputing the system Go version for debugging purposes.
-echo "System Go installation: `which go` is `go version`; Preparing to use $GOBIN/$GOVERS"
+echo "Preparing to use $GOBIN/$GOVERS"
```

Comment:
Any reason to cut this? It still feels moderately useful if anything goes wrong in this script and I don't think we would've printed it elsewhere

Reply:
Hopefully the earlier defensive checks prevent needing to know the installation anymore, but out of an abundance of caution, I've reinstated it.